libraries/api/package.json

```
@@ -9,7 +9,7 @@
     "coverage": "c8 yarn test",
     "coverage:report": "c8 yarn test && c8 report --temp-directory ./coverage/tmp --reporter json --report-dir ../../artifacts",
     "lint": "eslint src/*.js test/*.js",
-    "test": "mocha test/*_test.js"
+    "test": "export NODE_OPTIONS=--dns-result-order=ipv4first && mocha test/*_test.js"
```

Since Node.js v17, the DNS result order is returned how the resolver sent the list [PR](https://github.com/nodejs/node/pull/39987). This breaks some of our testing/infra scripts because IPv6 localhosts are now returned instead of IPv4.

---

```suggestion
    "test": "export NODE_OPTIONS=--enable-network-family-autoselection && mocha test/*_test.js"
```
see previous comment